app/assets/stylesheets/searchbar.scss

```diff
@@ -0,0 +1,48 @@
+
+/* ---------- SEARCH ---------- */
+
+.search-box {
+  text-align: center;
+  margin-top: 60px;
+  padding: 0;
+}
+
+#search {
+	background: transparent;
+	border-radius: 3px;
+	display: inline-block;
+	padding: 7px;
+}
+
+input#q.aaa-search-input {
+  width: 200px;
```

Comment:
Properties should be ordered border-bottom-left-radius, border-top-left-radius, height, width